package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "mathjs": "^7.5.1",
```

Comment:
Where do you use that library?

Reply:
I want use this library in eval function but it's not working